boards/nxp/frdm_mcxa153/Kconfig.defconfig

```diff
@@ -0,0 +1,11 @@
+# FRDM-MCXA153 board
+
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_FRDM_MCXA153
+
+config BOARD_EARLY_INIT_HOOK
+	default y
```

Comment:
Assuming that users should not be able to de-select this, needs to go to `Kconfig` file